src/devices/Ina226/Ina226Enum.cs

```diff
@@ -0,0 +1,249 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Iot.Device.Ina226
+{
+    /// <summary>
+    /// An enumeration representting the registers exposed by the INA226 device via I2c.
+    /// </summary>
+    internal enum Ina226Register : byte
```

Comment:
one type per file please (`ctrl + .` in VS should give you `Extract to ...` option)